android/java/org/chromium/chrome/browser/ntp_background_images/NTPBackgroundImagesBridge.java

```diff
@@ -1,4 +1,4 @@
-/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+2/* Copyright (c) 2020 The Brave Authors. All rights reserved.
```

Comment:
crap- my bad here. Will let Windows/macOS/Linux finish up and then will push a fix for this